### ISSUE TITLE
Camera permission text always display default text instead of custom.

### ISSUE THREAD

**Issue by novavision** [yasirkula/UnityNativeCamera#106]
**Description of the bug**

Camera permission text always display default text instead of custom.

**Reproduction steps**

So, by default it's set to "Allow {App} to take thee pictures and record video?". I changed it in Project Settings > yasirkula > Native Camera but nothing chnaged on both iOS and Android builds. I found a closed issue here where were recommended to change the text in `NCPostProcessBuild.cs`. So I did but it didn't solve the problem on iOS devices.

**Platform specs**

Please provide the following info if this is a Unity 3D repository.

- Unity version: 2021.3.23*
- Platform: Android
- Device: Xiaomi K20 Pro
- How did you download the plugin: Asset Store

**Comment by yasirkula** [comment]
Unity's Player Settings may be overriding those settings. Can you search Player Settings for that text to see if it exists?

**Comment by yasirkula** [comment]
Yes you can't change the text on Android. If they had added this option in the latest Android versions, then it's possible but I'm unaware of it.

**Comment by novavision** [comment]
Ok, that's clear, but why modified values (in Project Settings and script) are not applied to Android permission text? As I said I still see the default text there.

**Comment by yasirkula** [comment]
Android permission text can't be modified which is why those text aren't applied to it. The Android operating system shows a fixed text while asking for permission, you can't change the text.
